Travel-Planner/Travel-Planner/Views/TripListPageViews/CityCardView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  CityCardView.swift
+//  Travel-Planner
+//
+//  Created by siye yang on 10/3/24.
+//
+
+import SwiftUI
+
+struct CityCardView: View {
+    var cityName: String
+    var countryName: String
+    var imageName: String
+    
+    var body: some View {
+        ZStack(alignment: .bottomLeading) {
+            Image(imageName)
+                            .resizable() // Makes the image resizable
+                            .aspectRatio(contentMode: .fill) // Fills the card
+                            .frame(height: 150)
+                            .cornerRadius(15)
+                            .clipped() // Ensures the image fits within the bounds of the card
+            
+            RoundedRectangle(cornerRadius: 15)
```

Comment:
Any other way to do this?
you are using a zstack of 3 layers now

Reply:
I did another way using overlay